examples/igbh/dist_train_rgnn.py

```diff
@@ -258,6 +258,9 @@ def run_training_proc(local_proc_rank, num_nodes, node_rank, num_training_procs,
         if validation_acc is not None and global_acc >= validation_acc:
           is_success = True
           break
+        if with_gpu:
+          torch.cuda.synchronize()
+        torch.distributed.barrier()
```

Comment:
Currently there exists a synchronization step in the end of evaluation.  Any errors observed if the synchronization in line 161-163 is removed?

Reply:
Oh, Sorry, I didn't notice it. Will discard this.